LoopCore/NSUserDefaults.swift

```diff
@@ -180,6 +185,19 @@ extension UserDefaults {
         }
     }
     
+    public var lastProfileExpirationAlertDate: Date? {
+        get {
+            if let rawValue = object(forKey: Key.lastProfileExpirationAlertDate.rawValue) as? Date {
```

Comment:
Perhaps just `return object(forKey: Key.lastProfileExpirationAlertDate.rawValue) as? Date`? The if/else seems unnecessary.